### PR TITLE
Update web api usage to v4

### PR DESCRIPTION
**Pull request by davideberlein**
replace web api v3 calls with the v4 equivalent.

### CODE REVIEW

File: src/main/resources/static/application.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ class SampleApp {
     */
   async checkAuthentication () {
     const options = { credentials: 'include' }; // <1>
-    return await fetch(this.unbluBaseUrl + '/rest/v3/authenticator/isAuthenticated', options)
+    return await fetch(this.unbluBaseUrl + '/rest/v4/authenticator/isAuthenticated', options)
```
Comment (jmini) [inline src/main/resources/static/application.js]: I do not think that `authenticator/isAuthenticated` exists in v4

Comment (gysel) [reply src/main/resources/static/application.js]: At least it was removed from the API docs...

https://www.unblu.com/en/docs/latest/reference/unblu-web-api/#tag/Authenticator

**Review by jmini — CHANGES_REQUESTED** [review]
Please use v4 endpoints

File: src/main/resources/static/application.js (hunk 1 above)
Comment (davideberlein) [reply src/main/resources/static/application.js]: that's bad 😢 I knew I forget to do something... check if the v4 methods are still availible ;-)

**Review by gysel — APPROVED** [review]

**Review by jmini — APPROVED** [review]

**Pull request merged**